### ISSUE TITLE
Error Message/Code: Expected argument current_run to have a valid value.

### ISSUE THREAD

**Issue by mastreips** [Azure/azure-sdk-for-python#14463]
I am trying to modify the following notebook tutorial in the AML environment - /classification-text-dnn/auto-ml-classification-text-dnn.ipynb so that it outputs and saves the model in ONNX format.  I was able to save to pickle format, but when I try to save the output to ONNX using:

`automl_run.get_output(return_onnx_model=True)`

I get the following error:

ValidationException:
	Message: Encountered an internal AutoML error. Error Message/Code: Expected argument current_run to have a valid value.

I know that only certain models work with Onnx conversion through AutoML (reference: https://docs.microsoft.com/en-us/azure/machine-learning/how-to-configure-auto-train#configure-your-experiment-settings)  I attempted to prevent any compatibility issues via my AutoML settings below:

here is my AutoML config:

`automl_settings = {
    "experiment_timeout_minutes": 20,
    "experiment_exit_score": 0.70,
    "primary_metric": 'accuracy',
    "max_concurrent_iterations": num_nodes, 
    "max_cores_per_iteration": -1,
    "enable_dnn": False,
    "enable_early_stopping": True,
    "validation_size": 0.3,
    "verbosity": logging.INFO,
    "enable_voting_ensemble": False,
    "enable_stack_ensemble": False,
}

automl_config = AutoMLConfig(task = 'classification',
                             debug_log = 'automl_errors.log',
                             compute_target=compute_target,
                             training_data=train_dataset,
                             label_column_name=target_column_name,
                             blocked_models = ['LightGBM', 'Averaged Perceptron Classifier'],
                             enable_onnx_compatible_models=True,
                             **automl_settings
                            )`

**Comment by rtanase** [comment]
@mastreips, I believe no ONNX models were generated for the experiment. To be 100% sure, do you mind sharing the run id for which you tried to download the ONNX model?

**Comment by YunsongB** [comment]
@mastreips, we have identified this issue a couple of days before and released the fix in the azureml-automl-runtime version [1.16.0](https://pypi.org/project/azureml-automl-runtime/1.16.0/). Could you please upgrade the automl package(s) and retry?

**Comment by cartacioS** [comment]
Hi @mastreips can you please let us know if the fix above worked or if you are still facing the issue?

**Comment by cartacioS** [comment]
#please-close
